www/about.html

```diff
@@ -25,8 +25,71 @@
 
     <main>
       <a href="/">← Return to the tracker</a>
-      <h2>About</h2>
-      <p>Tracker is a concise and small application intended to help you get started with Hoodie. It's a basic tracker of all sorts. The mechanics let you keep track of anything you want plus its amount.</p>
+      <h2>What is Hoodie Tracker?</h2>
+
+      <p>
+        <img src="/assets/low-profile-dog.png" alt="wiggle waggle Low-Profile Dog" class="low-profile-dog">
```

Comment:
For the alt tag could we go the extra mile and be more descriptive?

'Illustration of Hoodie's Low-Profile Dog Mascot waving'